src/main/java/com/ubc/cpsc304/hotelify/controller/ReviewController.java

```diff
@@ -0,0 +1,52 @@
+package com.ubc.cpsc304.hotelify.controller;
+
+
+import com.ubc.cpsc304.hotelify.controller.dto.ReviewRequestDto;
+import com.ubc.cpsc304.hotelify.controller.dto.ReviewResponseDto;
+import com.ubc.cpsc304.hotelify.entity.Review;
+import com.ubc.cpsc304.hotelify.service.ReviewService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Objects;
+
+@Slf4j
+@RestController
+@RequestMapping("/reviews")
+public class ReviewController {
+    private final ReviewService reviewService;
+
+    @Autowired
+    public ReviewController(ReviewService reviewService) {
+        this.reviewService = reviewService;
+    }
+
+    @GetMapping
+    private String getReview() {
```

Comment:
this was just a skeleton method, you need to actually implement it, we probably also need a `getReviews` endpoint

Reply:
Pretty new here, not sure how spring works... give me some time to figure it out